app-k9mail/src/main/kotlin/app/k9mail/provider/K9AppNameProvider.kt

```diff
@@ -4,15 +4,26 @@ import android.content.Context
 import app.k9mail.core.common.provider.AppNameProvider
 import app.k9mail.core.common.provider.BrandNameProvider
 import com.fsck.k9.R
+import com.fsck.k9.preferences.FilePrefixProvider
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Locale
 
 class K9AppNameProvider(
     context: Context,
-) : AppNameProvider, BrandNameProvider {
+) : AppNameProvider, BrandNameProvider, FilePrefixProvider {
     override val appName: String by lazy {
         context.getString(R.string.app_name)
     }
 
     override val brandName: String by lazy {
         context.getString(R.string.app_name)
     }
+
+    override val filePrefix: String
+        get() {
+            val now = Calendar.getInstance()
+            val dateFormat = SimpleDateFormat("yyyy-MM-dd", Locale.US)
```

Comment:
This logic should stay in `SettingsExporter`. The prefix supplied by `FilePrefixProvider` should only be "k9" and "thunderbird".

Reply:
@cketti Done!